### ISSUE TITLE
Federated server component?

### ISSUE THREAD

**Issue by poVoq** [lutris/website#72]
Would it be possible to set up a kind of federated server backend with lutris?
I.e. similar to how XMPP can work with multiple servers with the same client and easy inter-operability.

**Comment by strycore** [comment]
It would be possible, just like everything is, but it would have more chances of happening if I had the slightest clue what this could be used for.

**Comment by poVoq** [comment]
It would lessen the dependence on a single point of failure and one could also distribute the server load for large files onto different people.

If a shop system was ever to be integrated, this could also allow companies to set up their own and seamlessly integrate with the lutris client software.

**Comment by charlydelta** [comment]
I don't see a use in this either. Plus, it would be an immense effort to implement this. The whole database would have to be reworked, for little to no benefit. I mean, who would federate? This issue is over a year old, and obviously there is no demand for such a feature.

And from what I can tell, I never experienced Lutris to be down (contrary to the games DB for example)

But you are free to setup your own server, of course. Everything is freely available.

So to sum it up:

- too much effort
- for little to no gain
- no demand
- lutris.net is reliable
- you can setup your own stack and secure it Form failure otherwise

I plea to close this issue

**Comment by strycore** [comment]
I agree. When someone does open another instance of the lutris.net website (for a real purpose, not just for the sake of having one online), then we might re-consider it but right now, he have more valuable things to focus on.
